packages/settings-library/src/settings_library/r_clone.py

```diff
@@ -15,3 +15,17 @@ class S3Provider(str, Enum):
 class RCloneSettings(BaseCustomSettings):
     R_CLONE_S3: S3Settings = Field(auto_default_from_env=True)
     R_CLONE_PROVIDER: S3Provider
+
+    # SEE https://rclone.org/docs/#transfers-n
+    R_CLONE_OPTION_TRANSFERS: NonNegativeInt = Field(
+        default=5, description="`--transfers X`: sets the amount of parallel transfers"
+    )
+    # SEE https://rclone.org/docs/#retries-int
+    R_CLONE_OPTION_RETRIES: NonNegativeInt = Field(
+        default=3, description="`--retries X`: times to retry each individual transfer"
+    )
+    # SEE https://rclone.org/docs/#buffer-size-size
+    R_CLONE_OPTION_BUFFER_SIZE: str = Field(
+        default="0M",
+        description="`--buffer-size X`: sets the amount of RAM to use for each individual transfer",
+    )
```

Comment:
be careful with this thing. do you account for it in the docker swarm RAM reservation? or how do you plan to account for it?

Reply:
so this is currently only done upon closing I agree. But please keep this in mind the day you do upload states at runtime... cause this might have an impact then.